onchain/cairo/src/launchpad/launchpad.cairo

```diff
@@ -1060,6 +1062,15 @@ mod LaunchpadMarketplace {
             let pool_coin = self.launched_coins.read(coin_address);
             let total_supply = pool_coin.total_supply.clone();
             let current_supply = pool_coin.token_holded.clone();
+            if (!is_decreased) {
```

Comment:
I think it's not the correct condition, this is the formula for the is_decreased == true no? 

Also, can you delete the parenthesis? Not needed in Rust/Cairo I think

Reply:
i see